lib/ui/auth_flow/create_profile_screen.dart

```diff
@@ -194,12 +194,15 @@ class _CreateProfileScreenState extends ConsumerState<CreateProfileScreen> {
           ),
         ),
       ),
-      bottomNavigationBar: SafeArea(
-        top: false,
-        child: Padding(
-          padding: EdgeInsets.symmetric(
-            horizontal: 24.w,
-          ).copyWith(bottom: 32.h),
+      bottomNavigationBar: Container(
+        padding: EdgeInsets.only(
+          left: 24.w,
+          right: 24.w,
+          top: 16.h,
+          bottom: 32.h + MediaQuery.of(context).viewInsets.bottom,
```

Comment:
The bottom padding seems too much. Also, a part of the "introduce yourself" textfield is still covered when keyboard is open.

https://github.com/user-attachments/assets/05413c1d-4c98-41ce-92c8-cf4b76c1fc3a